package.json

```diff
@@ -36,5 +36,8 @@
   ],
   "engines": {
     "node": ">=18.0.0 <21.0.0"
+  },
+  "devDependencies": {
+    "rollup-plugin-serve": "^1.1.1"
```

Comment:
Do Dev dependencies make sense at the mono-repo level, since the mono-repo is just for development?

Reply:
Ah you're right, I guess this can be a regular dependency.